src/NuGet.Core/NuGet.Build.Tasks/NuGet.targets

```diff
@@ -359,7 +359,7 @@ Copyright (c) .NET Foundation. All rights reserved.
     <CheckForDuplicateNuGetItemsTask
       Items="@(NuGetAuditSuppress)"
       ItemName="NuGetAuditSuppress"
-      LogCode="NU1505"
```

Comment:
@nkolev92 from https://github.com/NuGet/Home/issues/13620

> This should be a new log code.

Why do we use a different code for each MSBuild item type?

The docs are the same for each item type, since the resolution is the same. I feel like all this design decision does is cause us extra busy work.  I feel like one error code, where in the docs we mention that this can happen for multiple item types, would just be easier, with no downside.

Reply:
The owner of the action isn't always the same. 
In particular the SDK sets PackageDownload items, so any issues are likely problems on the SDK side. 

For example, if you search with NU1505, you end up with https://github.com/dotnet/sdk/issues/24747.
NU1504 gives you something else. 

It's duplicative sure, but I think there's an eventual limit to the number of items we'll be using :)